src/Memory/include/Memory/MemoryInfoProducer.h

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_MEMORY_INFO_PRODUCER_H_
```

Comment:
Include guard should be the name of the module followed by the name of the file.

Reply:
Changed accordingly. Thanks for pointing out this mistake. Please see the commit *[fixup] Add module name to the include guards*)